frontend/src/constants.js

```diff
@@ -21,6 +21,11 @@ export const EMAIL_REGEX = /^\S+@\S+\.\S+$/
 export const URI_MATCHING_SERVICE =

```

Comment:
Let's standardize to URL instead of URI, what do you think?

Reply:
Yeah sure, was just following `URI_USER_SERVICE`

Reply:
Ah ok sorry didn't realize, think the original template might be trying to differentiate stuff

Reply:
@MarcusTXK if there's no time we can merge this in first so that there's something to do for integration

Reply:
Will change in the future.